app/bot.go

```diff
@@ -360,7 +360,7 @@ func (a *App) UpdateBotActive(c *request.Context, botUserId string, active bool)
 }
 
 // PermanentDeleteBot permanently deletes a bot and its corresponding user.
-func (a *App) PermanentDeleteBot(botUserId string) *model.AppError {
+func (a *App) PermanentDeleteBot(c *request.Context, botUserId string) *model.AppError {
 	if err := a.Srv().Store.Bot().PermanentDelete(botUserId); err != nil {
```

Comment:
If `PermanentDeleteUser` is already calling `Bot().PermanentDelete`, can we remove this?
Also, if I am not mistaken, if there is not bot with that bot user id, this won't fail. So we could with this function Permanent delete any user. Should we add a check somewhere to make sure it is a bot? (e.g. `if !user.isBot {return error}`)

Reply:
Totally agree with that, we can remove the `Bot().PermanentDelete` and add a check it the user is a bot or not.

Anyway, I want to confirm with everybody if the `PermanentDeleteBot` current behavior is needed by someone, (specially playbooks).

Reply:
I like the idea of combining them. Probably shouldn't have been different in the first place. 
Playbooks isn't using this. Not aware of anything that does.

Reply:
I like the idea of combining them. Probably shouldn't have been different in the first place. 
Playbooks isn't using this. Not aware of anything that does.

Reply:
Done